### ISSUE TITLE
Accepted Introductions stick in .../notifications/system

### ISSUE THREAD

**Issue by AlfredSK** [friendica/friendica#8197]
### Expected behavior
When I accepted an introduction the introduction should vanish from all notification URLs or to say it more precisely should be marked as seen everywhere.

### Actual behavior
When I get a introduction I select the notification from the notification dropdown (bell icon). I accept the introduction successfully and the notification in the dropdown (bell icon) is marked as seen. Ok.

But there still is an entry for this intro in ... /notifications/system. Clicking on the link gives a nice themed error page (404) because there is no intro anymore.

The error page shows the stack trace in a small window on the left side of our mascot. Is this intended?

```
#0 [internal function]: Friendica\Module\HTTPException\PageNotFound::content(Array)
#1 /var/www/html/libranet/src/App/Page.php(310): call_user_func(Array, Array)
#2 /var/www/html/libranet/src/App/Page.php(364): Friendica\App\Page->initContent(Object(Friendica\App\Module), Object(Friendica\App\Mode))
#3 /var/www/html/libranet/src/App.php(583): Friendica\App\Page->run(Object(Friendica\App), Object(Friendica\App\BaseURL), Object(Friendica\App\Mode), Object(Friendica\App\Module), Object(Friendica\Core\L10n), Object(Friendica\Core\Config\PreloadConfig), Object(Friendica\Core\PConfig\PreloadPConfig))
#4 /var/www/html/libranet/index.php(27): Friendica\App->runFrontend(Object(Friendica\App\Module), Object(Friendica\App\Router), Object(Friendica\Core\PConfig\PreloadPConfig), Object(Friendica\App\Authentication), Object(Friendica\App\Page))
#5 {main}
```

### Steps to reproduce the problem
Get an introduction. Accept it as described. Go to .../notifications/system. Discover there is still an intro listed. Click on the link.

### Friendica version you encountered the problem
2020.03-dev

### Friendica source (git, zip)
git

### PHP version
7.2

### SQL version
10.1 MariaDB

**Comment by MrPetovan** [comment]
> The error page shows the stack trace in a small window on the left side of our mascot. Is this intended?

For admins, yes. The problem with the error web page is that it swallows log messages, but the trace is still pretty much crucial to debug.

What is the URL of the page not found error?

**Comment by AlfredSK** [comment]
> For admins, yes.

Ah! I thought everybody sees that. Ok.

The URL is probably https://friendica.server/notifications/intro. But I don't have a new introduction right now. So, I'm not absolutely sure.

**Comment by AlfredSK** [comment]
Btw. this behavior isn't new. It has been this way for month or even years.

**Comment by nupplaphil** [comment]
> Btw. this behavior isn't new. It has been this way for month or even years.

At least I'm now able to fix a bug which I didn't already introduced by myself .. yeah :D

**Comment by MrPetovan** [comment]
Oh, I thought it was a regression @nupplaphil had introduced, thanks for the clarification.

**Comment by nupplaphil** [comment]
Fixed with #8199
